### ISSUE TITLE
Outdated doc - --show-versions in status command

### ISSUE THREAD

**Issue by fey** [doctrine/migrations#1124]
Here: https://www.doctrine-project.org/projects/doctrine-migrations/en/3.0/reference/managing-migrations.html#managing-migrations

`--show-versions` was removed in this commit
https://github.com/doctrine/migrations/commit/be8275318a3983444d740e391de917ebdf8ed222

```
$ vendor/bin/doctrine-migrations status --show-versions
                                               
  The "--show-versions" option does not exist.  
```

**Comment by SenseException** [comment]
Good catch. D you like to create a PR to remove the option from the docs and adapt the output of `status`?

**Comment by fey** [comment]
Okay. I will try, but i get some problem to build docs locally.
So if somebody want to contribute - welcome

**Comment by SenseException** [comment]
You can push your dev-branch to Github to view your current changes or use a rst-viewer (e.g. the one in PHPStorm).

**Comment by rolandinsh** [comment]
Still in 3.7: https://www.doctrine-project.org/projects/doctrine-migrations/en/3.7/reference/managing-migrations.html

**Comment by greg0ire** [comment]
@rolandinsh would you like to create a PR to remove the option from the docs and adapt the output of `status`?
